Documentation/CRDs/ceph-client-crd.md

```diff
@@ -41,3 +41,146 @@ spec:
 ### Prerequisites
 
 This guide assumes you have created a Rook cluster as explained in the main [Quickstart guide](../Getting-Started/quickstart.md).
+
+## Examples
+
+### Using a `CephClient` to create Ceph credentials
+
+To use `CephClient` to connect to a Ceph cluster:
+
+#### 1. Create the `CephClient` (`metadata.name` will be your Ceph client ID)
+
+Create and apply a `CephClient` resource YAML:
+
+`example.cephclient.yaml`
+```yaml
+---
+apiVersion: ceph.rook.io/v1
+kind: CephClient
+metadata:
+  name: sqlitevfs
+  namespace: rook-ceph
+spec:
+  caps:
+    mon: 'allow r'
+    osd: 'profile rbd pool=example'
+```
+
+The configuration above allows read access to mons, and RBD access on all OSDs via RBD targeting pool `example`. You can apply this config with `kubectl`:
+
+```console
+kubectl apply -f example.cephclient.yaml
+```
+
+#### 2. Find the generated secret for the `CephClient`
+
+Once your `CephClient` has been processed by Rook (in the rook namespace, for example `rook-ceph`), it will be updated to include your secret:
+
+```console
+kubectl get cephclient example -o jsonpath='{.status.info.secretName}' --namespace rook-ceph
+```
+
+#### 3. Extract Ceph cluster credentials from the generated secret
+
+Extract Ceph cluster credentials from the generated secret (note that the subkey will be your original client name):
+
+```console
+kubectl --namespace rook-ceph get secret rook-ceph-client-example -o jsonpath="{.data.example}" | base64 -d
+```
+
+The base64 encoded value that is returned *is* the password for your ceph client.
+
+#### 4. Determine the location of mons for the cluster
+
+To send writes to the cluster, you must retrieve the mons in use:
+
+```console
+kubectl --namespace rook-ceph get configmap rook-ceph-mon-endpoints -o jsonpath='{.data.data}' | sed 's/.=//g'`
+```
+
+This command *should* produce a line that looks somewhat like this:
+
+```
+XXX.XXX.XXX.XXX:6789,YYY.YYY.YYY.YYY:6789,ZZZ.ZZZ.ZZZ.ZZZ:6789
```

Comment:
example IP addresses would be nice instead of the `xxx.xxx.xxx.xxx`

Reply:
- [ ] use example IP addresses instead of placeholders